src/main/groovy/org/nlp_uk/tools/TokenizeText.groovy

```diff
@@ -31,6 +31,8 @@ class TokenizeText {
     // easy way to include a class without forcing classpath to be set
     static textUtils = Eval.me(new File("$SCRIPT_DIR/TextUtils.groovy").text + "\n new TextUtils()")
 
+    def JSONUnicodeBuilder = new JsonGenerator.Options().disableUnicodeEscaping().build()
```

Comment:
In java/groovy vars start with lowercase, classes start with uppercase and constants all uppercase